relay.module.d.ts

```diff
@@ -17,22 +15,22 @@ export declare const UserIdHex: (userId: Uint8Array) => string;
 
 export declare class Relay {
   public ws: WebSocket;
-  public onClientDisconnect: ((userId: Uint8Array) => void) | null;
+  public onClientDisconnect: ((userId: Uint8Array) => unknown) | null;
   public onSubscription: ((userId: Uint8Array) => Relay) | null;
-  public BinaryMessageHandlers: { [key: keyof re.OP]: (sender: Uint8Array, message: Uint8Array | object) => void };
-  public TextMessageHandlers: { [key: keyof re.OP]: (sender: Uint8Array, message: Uint8Array | object) => void };
-  public VariableCallbacks: Function[];
+  public BinaryMessageHandlers: { [key: number]: (sender: Uint8Array, message: Uint8Array) => unknown };
+  public TextMessageHandlers: { [key: number]: (sender: Uint8Array, message: GenericObject) => unknown };
+  public VariableCallbacks: GenericFunction[];
   public channelName: string;
   public ready: boolean;
   public userId: Uint8Array;
 
   constructor(relayURL: string, channelName: string);
   public JoinChannel(): void;
-  public SetMessageHandler(msgType: number, OpCode: keyof re.OP, callback: unknown): void;
+  public SetMessageHandler(msgType: number, OpCode: number, callback: unknown): void;
   public MessageDispatcher(e: MessageEvent): void;
   public FirstMessageHandler(e: MessageEvent): void;
-  public bSendTo(target: Uint8Array, OpCode: keyof re.OP, msg: any): void;
+  public bSendTo(target: Uint8Array, OpCode: number, msg: any): void;
   public tSendTo(target: string | Uint8Array, obj: any): void;
   public SetChannelVar(key: string, value: any): void;
-  public GetChannelVar(key: string, callback: Function): void;
+  public GetChannelVar(key: string, callback: GenericFunction): void;
```

Comment:
Callbacks should have specific function types that match the args that the callbacks would take.  They shouldn't be generic because the types of functions you can pass isn't open-ended, it needs to be a function that takes the correct args.